### PR TITLE
Content changes

### PR DESCRIPTION
**Pull request by jlid87**

### CODE REVIEW

File: source/documentation/get-all-messages.html.md.erb [hunk 1]
```diff
@@ -5,25 +5,54 @@ weight: 60
...
+| Message | Schema link |
+| --- | --- |
+| Generic refusal message (IE704) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie704uk.xsd) |
+| E-AD/E-SAD container (IE801) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie801.xsd) |
+| Reminder Message for Excise Movement (IE802) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie802.xsd) |
+| Notification of Diverted E-AD/E-SAD (IE803) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie803.xsd) |
+| Interruption of Movement (IE807) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie807.xsd) |
+| Cancellation (IE810) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie810-v16-0.xsd) |
+| Change of destination (IE813) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie813-v16-0.xsd) |
+| Submit a new draft movement (IE815) | [XSD](https://github.com/hmrc/excise-movement-control-system-api/blob/main/app/xsd/ie815-v16-0.xsd) |
```
Comment (ChrisCookOC) [inline source/documentation/get-all-messages.html.md.erb:37]: The XML schema we have for Get New Messages from EMCS Core doesn't have IE815 on it as a valid response type?

Comment (jlid87) [reply source/documentation/get-all-messages.html.md.erb:37]: Correct, removing.

File: source/index.html.md.erb [hunk 2]
```diff
@@ -24,57 +22,57 @@ Use the EMCS API to:
...
 
-### First-time EMCS API Users
+### First-time Developer Hub Users
 
-If you have not previously used the EMCS API, you should:
+If you have not previously used the Developer Hub, you should:
 
-1. Ensure that you have an HMRC [developer account](https://developer.service.hmrc.gov.uk/developer/login) - if you do not have one, you must [register for an account](https://developer.service.hmrc.gov.uk/developer/registration), activate it by email, and sign in.
+1. [Sign up for an HMRC developer account](https://developer.service.hmrc.gov.uk/developer/registration), activate it by email, and [sign in](https://developer.service.hmrc.gov.uk/developer/login)).
 2. Add your subscription to this API to your application software.
```
Comment (ChrisCookOC) [inline source/index.html.md.erb:57]: this step applies to you if you have used the Developer Hub but not the EMCS API, so maybe needs to be in a separate section?

Comment (jlid87) [reply source/index.html.md.erb]: Retitled the section, moved some text around.

**Review by ChrisCookOC — APPROVED** [review]

**Review by ChrisCookOC — APPROVED** [review]

**Pull request merged**